src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -47,6 +47,10 @@ static ucs_config_field_t uct_cuda_ipc_iface_config_table[] = {
      "Max number of cuda events. -1 is infinite",
      ucs_offsetof(uct_cuda_ipc_iface_config_t, max_cuda_ipc_events), UCS_CONFIG_TYPE_UINT},
 
+    {"BW", "0MBs",
```

Comment:
pls use `auto` instead of `0`

Reply:
@brminich if we use `auto` would we be able to check [this](https://github.com/openucx/ucx/pull/8798/files#diff-04769c9662cb2e769aff68bed88f18cef6d6bdaa369b178d4223d769cecb7d4fR227) condition?

Reply:
yes, but need to compare with `UCS_CONFIG_DBL_AUTO` instead